### PR TITLE
work on invalid records

### PR DESCRIPTION
**Pull request by aab-odoo**

### CODE REVIEW

File: addons/web/static/tests/legacy/fields/relational_fields/field_one2many_tests.js [hunk 1]
```diff
@@ -8804,7 +8804,7 @@ QUnit.module('Legacy fields', {}, function () {
             assert.containsNone(form, '.some_button4');
 
             await testUtils.fields.many2one.clickOpenDropdown('product_id');
-            await testUtils.fields.many2one.clickHighlightedItem('product_id');
+            await testUtils.fields.many2one.clicgkHighlightedItem('product_id');
```
Comment (FrancoisGe) [inline addons/web/static/tests/legacy/fields/relational_fields/field_one2many_tests.js]: clickHighlightedItem ?

File: addons/web/static/tests/fields/one2many_field_tests.js [hunk 2]
```diff
@@ -10172,130 +10159,6 @@ QUnit.module("Fields", (hooks) => {
         assert.containsNone(form, ".some_button4");
     });
 
-    QUnit.skipWOWL(
```
Comment (FrancoisGe) [inline addons/web/static/tests/fields/one2many_field_tests.js:10175]: Remove ?

File: addons/web/static/tests/fields/one2many_field_tests.js [hunk 3]
```diff
@@ -10172,130 +10159,6 @@ QUnit.module("Fields", (hooks) => {
...
-            assert.containsN(
-                form,
-                "th:not(.o_list_record_remove_header)",
-                2,
-                "should be 2 columns('foo' + 'int_field')"
-            );
-        }
-    );
-
-    QUnit.skipWOWL("one2many column_invisible on view not inline", async function (assert) {
```
Comment (FrancoisGe) [inline addons/web/static/tests/fields/one2many_field_tests.js:10232]: remove ?

File: addons/web/static/tests/fields/one2many_field_tests.js (hunk 2 above)
Comment (aab-odoo) [reply addons/web/static/tests/fields/one2many_field_tests.js:10175]: those tests are duplicated, don't know why, probably a bad conflict resolution

**Pull request merged**